Core/Inc/can_handler.h

```diff
@@ -0,0 +1,27 @@
+#ifndef CAN_HANDLER_H
+#define CAN_HANDLER_H
+
+#include "can.h"
+#include "stm32f4xx_hal.h"
+#include <stdint.h>
+
+
+#define NUM_INBOUND_CAN1_IDS 1
+#define NUM_INBOUND_CAN2_IDS 1
+
+static const uint16_t can1_id_list[NUM_INBOUND_CAN1_IDS] = {
+	//CANID_X,
+	0x0000
+};
+
+static const uint16_t can2_id_list[NUM_INBOUND_CAN2_IDS] = {
+	//CANID_X,
+	0x0000
+};
+
+/* Shepherd hanbdles both can busses the same way */
+void can_receive_callback(CAN_HandleTypeDef *hcan);
```

Comment:
tbh i was indifferent, my logic was that from shepherds POV it doesnt care where messages are coming from, it needs to read and process them the same regardless. Addressing concerns is valid i guess, i sort of thought our convo about CAN ids implied wed maintain our structure across busses, so the callback could recognize CAN ID indifferent of bus its received. if not the case i will adjust

Reply:
I think that is fair but then we'd have to worry about ID conflicts between charger + everything else. We can consider it just an extension of the CAN ID framework we made tho if there isn't any problem but we'd need to allocate another static section since the charger is OTS. Also keep in mind that there is also an energy meter on the 500kbps bus when the car is actually active which we need to look into ID conflicts for that too. IMO its easier to be super deliberate about IDs on car and then for the second CAN bus just kinda shit out IDs wherever since the amount of unique messages is less

Reply:
yeah thats fine i will make second cb